tests/integration/test_fim/test_files/test_audit/data/ossec.conf

```diff
@@ -291,4 +291,4 @@
     <disabled>yes</disabled>
   </cluster>
 
-</ossec_config>
\ No newline at end of file
+</wazuh_config>
```

Comment:
This is legacy, but we should add the trailing EOL here.

Reply:
Done, thank you!